### DISCUSSION TITLE
Mismatched Inference Speed with Coral + Synology + Docker

### DISCUSSION THREAD

**Discussion by russelltomkins** [blakeblackshear/frigate#8710] [Ask A Question] [answered]
My second coral TPU arrived today and I plugged it into Synology DS1621+ with Docker and I noticed I'm getting mismatched inference speeds between the two TPU's. The second one is the new unit.

Does frigate "balance" it's work effort between the TPU's or is there some investigating I need to do with the first one as I believe it should be lower than the speed shown. The first one has always been ~20ms. There are currently 5 cameras monitored.

![image](https://github.com/blakeblackshear/frigate/assets/15333641/1e889bd9-81f3-4eea-8df2-633479224172)

**Comment by NickM-27** [comment]
This is usually down to a hardware difference like USB port or one USB port shares a BUS with something else.

**Reply by russelltomkins** [reply, marked as answer]
Each device has it's own dedicated USB Hub/Port, BUT .... I did notice the cable on the slower one was nowhere near as thick/high-quality as the brand new one. Swapped it out for a better quality one (still not the original one though) and things are nice and snappy on both of them now 😀 

![image](https://github.com/blakeblackshear/frigate/assets/15333641/692f423b-227d-42fe-bf50-92bf5df9ab23)
